server/e2e/src/main/kotlin/org/jetbrains/bsp/bazel/BazelBspSampleRepoTest.kt

```diff
@@ -586,7 +586,9 @@ object BazelBspSampleRepoTest : BazelBspTestBaseScenario() {
     val targetWithDependencyJavaBinary =
       DependencySourcesItem(
         BuildTargetIdentifier("$targetPrefix//target_with_dependency:java_binary"),
-        emptyList(),
+        listOf(
+          "file://\$BAZEL_OUTPUT_BASE_PATH/external/guava/guava-28.0-jre-src.jar",
```

Comment:
Should we maybe keep this change and the change in `BspRequestsRunner.kt`? Those don't seem to blame for the issue we're experiencing, but I want to make sure because I don't know exactly what they do.

Reply:
This is a test, I assumed that it would fail without this, but I didn't read it closely enough to make sure. The `BspRequestRunner.kt` change just adds a helpful log, but it doesn't change the actual behavior of the build server.

Reply:
Could we verify that the test still passes? Also, we might want to keep the logging change.

Reply:
It looks like we broke that test (and possibly a bunch more) when we added the support for the lucidsoftware rules scala, so I'm not sure if this particular change makes a difference or not.

Reply:
If we want to make this mergeable, it'd be good to fix the tests, but
1. The test failures already exist in the branch I'm trying to merge into
2. I have a sinking feeling this was removed for performance reasons, so I don't know that jetbrains will let us merge this anyways since it would make intellij worse.